src/shared/components/VideoPlayer/VideoPlayer.style.tsx

```diff
@@ -42,12 +42,18 @@ export const Container = styled.div<ContainerProps>`
 
   .vjs-control-bar {
     font-family: ${typography.fonts.base};
-    background-color: rgba(0, 0, 0, 0.3);
-    height: ${sizes(16)} !important;
+    background: none;
+    margin-top: auto;
+    padding: 0;
+    z-index: ${zIndex.overlay + 1};
     align-items: center;
-
     /* account for progress bar on top */
```

Comment:
I think comment doesn't apply here anymore? We should probably move it along with `padding` prop